.github/workflows/build-wasi-nn-hip.yml

```diff
@@ -0,0 +1,33 @@
+name: Build WASI-NN ggml (HIP)
```

Comment:
Please follow the same way of how we handle the plugin workflows:
1. Add your extensions here: https://github.com/WasmEdge/WasmEdge/blob/master/.github/workflows/matrix-extensions.json
2. Modify the jobs to add your own dependencies: https://github.com/WasmEdge/WasmEdge/blob/master/.github/workflows/reusable-build-extensions-on-linux.yml